database/models/core.py

```diff
@@ -140,6 +140,7 @@ class Repository(CodecovBaseModel):
             random.choice(string.ascii_uppercase + string.digits) for _ in range(10)
         ),
     )
+    upload_token = Column(postgresql.UUID)
```

Comment:
Nothing crazy here, we've had this field in postgres for a while, just never tracked it in worker